polyA/converters/alignment_to_stockholm.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+# given an alignment file and alignment tool, call the correct stockholm parser
+
+if [ $# -lt 2 ]
+then
+  echo "$0: takes 2 arguments: alignment file, alignment tool" 1>&2
+  exit 1
+fi
+
+alignment_file="$1"
+alignment_tool="$2"
+
+if [ "$alignment_tool" = "cross_match" ]; then
+  python cm_to_stockholm.py "$alignment_file"
+fi
```

Comment:
We also have a RM converter, right? The shell script probably ought to live outside the Python package, also.

Reply:
Should move the CM converter too? So the converters directory (outside of the code dir) would have the RM converter, the CM converter, and this script

Reply:
Yeah, let's do that. I don't remember why the Stockholm converter ended up inside the module and we might as well have them all live together.